tests/voq/test_fabric_reach.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import yaml
+
+logger = logging.getLogger(__name__)
+# This test only runs on t2 systems.
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+supReferenceData = {}
+localModule = 0
+supervisorAsicBase = 300
+
+keys = []
+# There are 12 asic on Supervisor now.
+# Initialize the reference data dictionary for sup.
+num_asics = 12
+for i in range(num_asics):
+    keys.append('asic' + str(i))
+supReferenceData = {key: {} for key in keys}
+
+# This test checks the output of the "show fabric reachability" command
+# on one linecard. It is called once for each linecard in the chassis.
+# It loads the reference data for the linecard, runs the CLI command,
+# and compares the output.
+
+
+def test_fabric_reach_linecards(duthosts, enum_frontend_dut_hostname):
```

Comment:
can we add a check for voq here or in conditional_marker to skip for no voq platforms?

Reply:
Thank you for your comments. I updated the check in conditional_marker.